src/TimerOutput.jl

```diff
@@ -5,15 +5,17 @@ mutable struct TimeData
     ncalls::Int
     time::Int64
     allocs::Int64
+    firstexec::Int64
 end
-
+TimeData(ncalls, time, allocs) = TimeData(ncalls, time, allocs, time)
 Base.copy(td::TimeData) = TimeData(td.ncalls, td.time, td.allocs)
 TimeData() = TimeData(0, 0, 0)
```

Comment:
To fix the tests, do this
```suggestion
TimeData() = TimeData(0, 0, 0, time_ns())
```

Reply:
Thanks!